setup.py

```diff
@@ -27,7 +27,9 @@
         setup_requires=['setuptools_scm'],
         install_requires=[
             "fs~=2.0",
-            "google-cloud-storage~=1.0"
+            "google-cloud-storage~=1.0",
+            "urllib3~=1.25",
+            "packaging~=20.0"
```

Comment:
Why do we require this as an explicit runtime dependency?

Reply:
Because it is needed during runtime for version checking of the `urllib3` dependency. More specifically we use the `parse_version` function from `packaging`.

These are the core packaging utilities for python packages and it is maintained by the Python Packaging Authority...